backend/core/detection.py

```diff
@@ -768,7 +780,6 @@ def commit_hijack(
 
             if "hij_key" in monitor_event:
                 monitor_event["final_redis_hijack_key"] = redis_hijack_key
-                return
```

Comment:
this is critical; now even for hijack BGP updates that are reconsidered we proceed to re-detection, since their keys and context have changed.